pyrightconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+    "typeCheckingMode": "basic",
```

Comment:
I guess a worry with this is that I think we may want different modes for different parts of astropy...

Reply:
We *could* have more than one configuration file if it comes to that, but it's a long shot.